frontend/test/account-send.test.ts

```diff
@@ -185,27 +193,33 @@ describe('decryptData', () => {
     {
       advancedMode: true,
       usePublicKeyChecked: true,
-      encryptionCount: 0,
+      encryptionCount: '0',
       expectedCiphertext: '0x9f3873e440b439d4e7561e70b5db28af7abb67257f6353e8d6e057bd2ed16621',
     },
     {
       advancedMode: false,
       usePublicKeyChecked: false,
-      encryptionCount: 1,
+      encryptionCount: '1',
       expectedCiphertext: '0xac7a4c827e636bb25fbf1ae604c48f0c035c34f526456dc6264b5ee43e3119f2',
     },
     {
       advancedMode: true,
       usePublicKeyChecked: false,
-      encryptionCount: 2,
+      encryptionCount: '2',
       expectedCiphertext: '0xb609426a8909759990b22756b7f2ce4f8d5ac4685d6e2c40daa830d059950504',
     },
     {
       advancedMode: false,
       usePublicKeyChecked: true,
-      encryptionCount: 3,
+      encryptionCount: '3',
       expectedCiphertext: '0xd45ffb2b6d4b4ad3bc0682b111cdbceecac938df0c72a0b0a79f7b6be6cfeb3e',
     },
+    {
+      advancedMode: false,
+      usePublicKeyChecked: true,
+      encryptionCount: ethers.constants.MaxUint256.toString(),
```

Comment:
Based on how we generate the random count, this is not a possible encryption count. We are technically generating a max private key, which has a max value a bit smaller than MaxUint256

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/494/commits/fd8d6811a2021d67f3b5cab2057062ad46a07f85